wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -7,13 +7,14 @@
 */
 defined('ABSPATH') or die('No script kiddies please!');
 
-include 'abAlternateClass.php';
-include 'abContentAltererSettingsPage.php';
-
+spl_autoload_register(function($className){
+  $className = strtolower(str_ireplace("ABContentPlugin\\", "", $className));
+  include "classes/" . $className . ".php";
+});
 
 
 if (is_admin()) {
-  new abContentAltererSettingsPage();
+  new ABContentPlugin\SettingsPage();
 }

```

Comment:
Shouldn't 'else' be here? You don't need to initiate content alteration in the backend.